src/server/auth/server/testing/admin_test.go

```diff
@@ -262,116 +436,89 @@ func TestAdminFixBrokenRepo(t *testing.T) {
 	require.Equal(t, 1, CommitCnt(t, adminClient, repo)) // check that a new commit was created
 }
 
-// TestModifyAdminsErrorMissingAdmin tests that trying to remove a nonexistant
-// admin returns an error
-func TestModifyAdminsErrorMissingAdmin(t *testing.T) {
```

Comment:
Just noting here for posterity that this (removing all roles from an admin that has none to begin with) is no longer an error